Fixing some annoying bugs.

labkey-danield opened the pull request:
#### Rationale
Fixed two issues in common test code.

#### Related Pull Requests
* None

#### Changes
* Fixed issues in createUser that fails creation because of case differences in email address.
* Added a scrollIntoView to fix issue when grid rows are scrolled under the grid header and cannot be clicked.

labkey-danield committed:
AddToEnd test.
Add wait after clicking save.
Change APIUserHelper.createUser to be case insensitive when validate email address.

labkey-danield committed:
Merge branch 'develop' into fb_testAddBox

labkey-danield committed:
Merge branch 'develop' into fb_testAddBox

labkey-danield committed:
Merge branch 'develop' into fb_testAddBox

labkey-danield committed:
Merge branch 'develop' into fb_testAddBox

labkey-danield committed:
Merge branch 'develop' into fb_testAddBox

labkey-danield committed:
Merge branch 'develop' into fb_testAddBox

labkey-danield committed:
See if scrolling the checkbox into view will fix the error on TeamCity.

labkey-tchad reviewed (changes requested):
I would leave this as is. The server converts emails to lower-case and, generally, accepts either case but our test helpers do not. This check ensures that the email used to create the user matches what will actually show up in the UI and in `email` fields of various APIs.

labkey-danield committed:
Merge branch 'develop' into fb_testAddBox

labkey-danield committed:
Merge branch 'develop' into fb_testAddBox

labkey-danield committed:
Remove unnecessary test (addToEnd).
Add test to check for default storageType.
Use better code to build validation path to new storage location.
Remove case insensitive check of use email address during user creation.
Remove to address grid row being under grid header and not being clickable.

labkey-danield committed:
Merge branch 'develop' into fb_testAddBox

labkey-danield commented:
I'm assuming that because this comment was made globally and not on the individual file the ask was not to change either file.

labkey-tchad reviewed (approved):
Looks like somebody else already added that scroll and it got merged in to this branch. This is just an import cleanup now.

labkey-danield committed:
Changed FMAddBoxToFreezerPanel.clickSaveExpectError to throw if no error is shown.
Add back the scroll into view for GridRow.select.
